services/analytics/metrics_names_test.go

```diff
@@ -0,0 +1,68 @@
+// qan-api
+// Copyright (C) 2019 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package analitycs
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/Percona-Lab/qan-api/models"
+	"github.com/percona/pmm/api/qanpb"
```

Comment:
We typically split this in 3 groups:
1. stdlib imports.
2. Third-party imports (including imports from other Percona repos).
3. Imports from this repo.